test/test_codegen.py

```diff
@@ -68,13 +68,15 @@ def _pyro_make_array(self, res_list):
             if all_numbers:
                 return self.usr_np.array(res_list, dtype=self.usr_np.float64)
 
-            result = self.usr_np.empty_like(res_list, dtype=object,
-                                            shape=(len(res_list),))
+            # Jax does not support object arrays
+            # result = self.usr_np.empty_like(res_list, dtype=object,
+            #                                 shape=(len(res_list),))
```

Comment:
```suggestion
```
I've grown fond of just removing stuff like this.  You can always bring it back with git if you really need.

Reply:
You're right @MTCam, thanks!